charts/redpanda/values.schema.json

```diff
@@ -982,6 +982,23 @@
         "initContainers": {
           "type": "object",
           "properties": {
+            "fsValidator": {
+              "type": "object",
+              "properties": {
+                "enabled": {
+                  "type": "boolean"
+                },
+                "expectedFS": {
+                  "type": "string"
+                },
+                "resources": {
+                  "type": "object"
+                },
+                "extraVolumeMounts": {
+                  "type": "string"
+                }
```

Comment:
Should this be type string or object?

Reply:
Yes good question. Maybe a bad choice from me, I was considering this to be an array for but after looking at a few existing charts with prior art it seems that the choice was to capture this as a string. Maybe this is too confusing.